app/models/measure_component.rb

```diff
@@ -58,6 +58,18 @@ def ad_valorem?
       monetary_unit_code.nil?
   end
 
+  def expresses_unit?
+    measurement_unit_code
```

Comment:
is truthy right here? could it end up as a blank string?

or should it be `.present?`

Reply:
Not required from my experience of measure components previously, no